drivers/display/display_renesas_lcdc.c

```diff
@@ -515,6 +511,29 @@ static int display_smartbond_write(const struct device *dev,
 	return 0;
 }
 
+#if defined(CONFIG_PM_DEVICE) || defined(CONFIG_PM_DEVICE_RUNTIME)
+static int display_smartbond_pm_action(const struct device *dev, enum pm_device_action action)
```

Comment:
Not sure I understand the idea here- is the reasoning that applications should be able to defer powering on the LCDC, and then be able to do so using `pm_device_runtime_get`? Since an app would know when (if ever) it was ok with blanking the display and disabling the LCDC by calling `pm_device_runtime_put`, shouldn't the LCDC permit powering off via this method?

Reply:
Correct, but this approach should be valid only when PM_DEVICE_RUNTIME is used. When PM_DEVICE is used there is no way to turn on/off the LCDC explicitly, right?

Reply:
> Correct, but this approach should be valid only when PM_DEVICE_RUNTIME is used

Then maybe we need different handling for `PM_DEVICE_RUNTIME`? I guess my core point here is that runtime device PM should provide relatively granular control for the application over what is powered on and what isn't

> That is, the LCDC can be turned off upon calling display_blanking_on and be turned on when invoking display_blanking_off.

This would be a good way to determine when disabling the controller is allowed, yeah